libimage/runtime.go

```diff
@@ -541,6 +541,9 @@ type RemoveImagesOptions struct {
 	// using a removed image.  Use RemoveContainerFunc for a custom logic.
 	// If set, all child images will be removed as well.
 	Force bool
+	// LookupManifest ensures that {lookupManifest: true} it passed as opt to lookupImages
+	// which ensures that only valid manifestList is removed from store and no referenced images.
```

Comment:
Exposing internals (i.e., `lookupManifest`) in a public API isn't necessarily helpful to users.  Rather the comment should explain what it does on a higher, user-visible level.

Reply:
@vrothberg fixed comments also added another option `ManifestList` which we recently added since we are going to use it for `rmi` cases.

Reply:
Before adding more things, can you point to the issue we need to solve?

Reply:
@vrothberg `buildah manifest rm <name>` should only remove manifest list and no referenced image so ImageLookup is called with option `lookupManifest: true`.

However `buildah rmi <name>` can remove manifest list if found otherwise it should search for images so we call ImageLookup with `ManifestList: true`

Reply:
Let me check something quickly, I don't think we need that if we look up by ID.